src/hooks/useAppRepos.js

```diff
@@ -22,12 +22,36 @@ function useAppRepos() {
       for (let index = 0; index < apps.length; index++) {
         try {
           const app = apps[index]
-          const repo = await getApmRepo(
-            app.appName,
-            'latest',
-            getNetworkType(),
-            ethers
-          )
+
+          let repo
+
+          // If there are multiple published apps, find the first one.
+          // TODO: Support installing of all different versions.
+          if (Array.isArray(app.appName)) {
```

Comment:
Maybe we can make every `appName` an array on the config file so we can always expect it to be an array and handle a single case.